synapse/lib/stormhttp.py

```diff
@@ -554,7 +554,7 @@ def getObjLocals(self):
             'msgpack': self._httpRespMsgpack,
         }
 
-    async def _httpRespJson(self, encoding=None, errors='surrogatepass'):
+    async def _httpRespJson(self, encoding=None, errors='strict'):
```

Comment:
Does this still need to be adjustable?

Reply:
Adjusted to a `strict` param with default=False